js/repl/src/PluginConfig.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import type { PluginConfig } from './types';
+
+const pluginConfigs: Array<PluginConfig> = [
+  {
+    baseUrl: 'https://unpkg.com',
+    label: 'Minify (Babili)',
+    package: 'babili-standalone',
+    version: '0'
+  },
+  {
+    label: 'Prettify',
+    package: 'prettier',
+    version: '1'
+  }
+];
+
+const presetPluginConfigs: Array<PluginConfig> = [
+  /* TODO Add support for preset-env
+  {
+    label: 'env',
+    package: 'babel-preset-env',
+    version: '1'
+  },
+  */
+  {
+    label: 'es2015',
+    package: 'babel-preset-es2015',
+    version: '6'
+  },
+  {
+    label: 'es2015-loose',
+    package: 'babel-preset-es2015-loose',
+    version: '7'
```

Comment:
Just for my own understanding, these version numbers here currently don't do anything since we pull it from `babel-standalone`, yea?

(Noticed this was 7 where the rest were 6)